.ci_support/win64.yaml

```diff
@@ -4,5 +4,6 @@ cgo_compiler:
   - go-cgo
 channel_sources:
   - conda-forge
+  - pytorch
```

Comment:
Don't touch this file. Recipes on conda-forge should only depend on packages in conda-forge.

Reply:
Thanks @xhochy, updated